### PR TITLE
Added text to index.html

### PR DESCRIPTION
**Pull request by enzosromano**

### CODE REVIEW

**Review by G00SE-Y — APPROVED** [review]
Nice !

File: src/views/index.html [hunk 1]
```diff
@@ -22,6 +22,6 @@
 
     <h1>Hello!</h1>
```
Comment (IFlowLikeH2O) [inline src/views/index.html:23]: good

Comment (enzosromano) [reply src/views/index.html:23]: this code actually doesnt suck because

Comment (enzosromano) [reply src/views/index.html:23]: i will fix it

**Review by IFlowLikeH2O — APPROVED** [review]
Best

File: src/views/index.html (hunk 1 above)
Comment (G00SE-Y) [reply src/views/index.html:23]: you better

**Pull request merged**